src/main/resources/application.properties

```diff
@@ -15,14 +15,15 @@ logging.level.com.gw=FATAL
 logging.level.org.hibernate=FATAL
 
 
+spring.config.import=optional:file:../application.properties
```

Comment:
How do you know `application.properties` will exist under `..`? What if people run the `java -jar` command from some random path?